roles/splunk_common/tasks/main.yml

```diff
@@ -87,6 +87,21 @@
     - "'svc_port' in splunk"
     - splunk.svc_port | int != 8089
 
+- include_tasks: set_appserver_port.yml
+  when:
+    - "'appserver_port' in splunk"
+    - splunk.appserver_port | int != 8065
+
+- include_tasks: set_kvstore_port.yml
+  when:
+    - "'kvstore_port' in splunk"
+    - splunk.kvstore_port | int != 8191
+
+- include_tasks: set_hec_port.yml
```

Comment:
I believe my PR should take care of the HEC port: https://github.com/splunk/splunk-ansible/pull/402

Reply:
This is actually a good catch, After I deleted `set_hec_port.yml`, I forgot to delete the task. Thanks, removed.